shipyard_blueprints/googlecloud/shipyard_googlecloud/utils.py

```diff
@@ -65,13 +66,13 @@ def upload_file(bucket, source_full_path, destination_full_path):
         ) from e
 
 
-def get_gclient(credentials: str):
+def get_gclient(credentials):
     """
     Attempts to create the Google Cloud Storage Client with the associated
     environment variables
     """
     try:
-        return storage.Client()
+        return storage.Client(credentials=credentials)
```

Comment:
I think it would be cleaner  to turn `get_credentials` to a private function and call it within the `get_client` function mainly because we are leveraging the google sdk 
```
def get gclient():
      try:
           return storage.Client(credentials = _get_credentials())
      except ExitCodeException:
           raise
      except Exception as e:
           ...